bazel/swift_static_framework.bzl

```diff
@@ -162,7 +162,8 @@ def swift_static_framework(
     module_name = module_name or name + "_framework"
     if objc_includes:
         locations = ["$(location {})".format(x) for x in objc_includes]
-        copts = copts + ["-import-objc-header"] + locations
+        for location in locations:
+            copts = copts + ["-import-objc-header", location]
```

Comment:
This flag only works once, and swift picks the last one https://github.com/apple/swift/blob/e7c294a076b41a9c87be99bf28d91f864977012a/lib/Frontend/CompilerInvocation.cpp#L582-L583

Reply:
Thanks for spotting this @keith! That explains some of the weird ordering errors I was seeing too. I will update this rule subsequently to generate a bridge header from all objc-includes.